hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/state/LiveState.java

```diff
@@ -61,6 +64,7 @@ public void removeRunningTask(Protos.TaskID taskId) {
     runningTasks.remove(taskId.getValue());
   }
 
```

Comment:
Ok, if there is a plan to fix this soon, that is okay by me. Please add a todo with your name then